plaid/src/apis/npm/npm_web_client.rs

```diff
@@ -483,12 +491,13 @@ impl Npm {
             .map_err(|_| ApiError::NpmError(NpmError::LoginFlowError))?;
         let params: InviteUserToOrganizationParams =
             serde_json::from_str(params).map_err(|_| ApiError::BadRequest)?;
-        let team = params.team.unwrap_or(plaid_stl::npm::NpmTeam::Developers);
+        let user = self.validate_npm_username(&params.user)?;
+        let team = params.team.unwrap_or("developers".to_string());
```

Comment:
This is a pretty strong default. Are we sure that it's the right one (generally speaking, for any org using this)?

Reply:
If you're confident, just leave a comment explaining why this is a default and you can resolve this comment.

Reply:
Right, I will add a comment pointing to https://docs.npmjs.com/about-developers-team